mubench.reviewsite/tests/CSVHelperTest.php

```diff
@@ -0,0 +1,366 @@
+<?php
+
+use Monolog\Logger;
+use PHPUnit\Framework\TestCase;
+use MuBench\ReviewSite\Model\ExperimentResult;
+use MuBench\ReviewSite\Model\DetectorResult;
+use MuBench\ReviewSite\Model\Detector;
+use MuBench\ReviewSite\Model\Misuse;
+use MuBench\ReviewSite\Model\Review;
+
+use \MuBench\ReviewSite\CSVHelper;
+
+class CSVHelperTest extends TestCase
+{
+
+    private $csv_helper;
+
+    function setUp()
+    {
+        $this->csv_helper = new CSVHelper();
+    }
+
+    public function test_csv_from_stats_ex1()
+    {
+        $stats = [];
+        $stats[1] = new DetectorResult(
+            new Detector("-d1-", 1)
+            , [[
+            "exp" => "ex1",
+            "project" => "-p1-",
+            "version" => "-v-",
+            "result" => "success",
+            "runtime" => "42.1",
+            "number_of_findings" => "23",
+            "misuses" => [new Misuse(
+                ["misuse" => "0", "snippets" => [0 => ["line" => "5", "snippet" => "-code-", "id" => "1"]]],
```

Comment:
Is this fixture really minimal? I don't think snippets appear anywhere in the output, do they?